dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/AbstractLoadBalance.java

```diff
@@ -75,22 +73,17 @@ static int calculateWarmupWeight(int uptime, int warmup, int weight) {
     protected int getWeight(Invoker<?> invoker, Invocation invocation) {
         int weight;
         URL url = invoker.getUrl();
-        // Multiple registry scenario, load balance among multiple registries.
-        if (REGISTRY_SERVICE_REFERENCE_PATH.equals(url.getServiceInterface())) {
```

Comment:
`REGISTRY_KEY + "." + WEIGHT_KEY` needs to be reserved.
Registry's weight is not equals with provider's weight. Also, registry's weight should no override provider's weight.